locales/fr/messages.properties

```diff
@@ -0,0 +1,223 @@
+##########################################
```

Comment:
Ha I'm pretty sure this was because we were working with this file manually mid-way through the l10n process.
You can git checkout this file!